docs/gathered-data.md

```diff
@@ -903,6 +903,37 @@ gathers the alerts that have been silenced.
   * 4.10+
 
 
+## StorageCluster
+
+Collects `storageclusters.ocs.openshift.io` resources
+
+### API Reference
+- https://github.com/red-hat-storage/ocs-operator/blob/main/api/v1/storagecluster_types.go
+
+### Sample data
+- docs/insights-archive-sample/config/storage/openshift-storage/storageclusters/ocs-storagecluster.json
+
+| Version   | Path														|
+| --------- | --------------------------------------------------------	|
+| < 4.12.0  | config/storage/{namespace}/{name}.json 					|
+| >= 4.12.0 | config/storage/{namespace}/storageclusters/{name}.json 	|
+
+### Config ID
+`clusterconfig/storage_cluster`
+
+### Released version
+- 4.11.0
```

Comment:
I could be wrong but I think the agreement was that we wouldn't update this docs format in previous versions.  This is currently different structure than `## CephCluster` docs. I would use the original structure format.

Reply:
It should be ok now.

Reply:
Thank you.